components/tools/OmeroPy/src/omero/plugins/web.py

```diff
@@ -82,15 +84,18 @@ def _configure(self, parser):
 
         config = parser.add(
             sub, self.config,
-            "Output a config template for server"
-            " ('nginx' or 'apache' for the moment)")
-        config.add_argument("type", choices=("nginx", "apache"))
+            "Output a config template for web server\n"
+            "  nginx: Nginx system configuration for inclusion\n"
+            "  nginx-development: Standalone user-run Nginx server\n"
+            "  apache: Apache 2.2 with mod_fastcgi\n"
+            "  apache-fcgi: Apache 2.4+ with mod_proxy_fcgi\n")
+        config.add_argument("type", choices=(
+            "nginx", "nginx-development", "apache", "apache-fcgi"))
         config.add_argument(
             "--http", type=int,
             help="HTTP port for web server (not fastcgi)")
         config.add_argument(
-            "--system", action="store_true",
```

Comment:
You'll know better than I. Was mostly just the hope of forcing the fewest number of people to change their installation scripts.